gwt-timer/src/main/java/org/gwtproject/timer/client/Timer.java

```diff
@@ -26,8 +26,8 @@
  *
  * <p>NOTE: If you are using a timer to schedule a UI animation, use
  * <b>org.gwtproject.animation.client.AnimationScheduler</b> or
- * <b>DomGlobal#requestAnimationFrame</b>
- * instead. The browser can optimize your animation for maximum performance.
+ * <b>DomGlobal#requestAnimationFrame</b> instead. The browser can optimize your animation for
```

Comment:
Probably fix this line break, maybe newline after `optimize`

Reply:
Think, we can not  change that, cause the fmt-m-p does the line berak. If we change it, we will get a format error. :(